GetIntoTeachingApi/Models/SchoolsExperience/SchoolsExperienceSignUp.cs

```diff
@@ -66,7 +66,12 @@ private void PopulateWithCandidate(Candidate candidate)
             AddressCity = candidate.AddressCity;
             AddressStateOrProvince = candidate.AddressStateOrProvince;
             AddressPostcode = candidate.AddressPostcode;
-            Telephone = candidate.Telephone.StripExitCode();
+
+            // We only want to expose primary phone number to School Experience. However, there
+            // are a number of records in the CRM that have School Experience but no primary number.
+            // Fetch the other phone numbers for now until the issue has been resolved in the CRM.
+            var telephoneReserves = new string[] { candidate.MobileTelephone, candidate.AddressTelephone, candidate.SecondaryTelephone };
+            Telephone = candidate.Telephone.StripExitCode() ?? Array.Find(telephoneReserves, number => !string.IsNullOrWhiteSpace(number));
```

Comment:
Would we not want to call `StripExitCode()` on the result of this expression, or is it intentional it only applies to `candidate.Telephone` and not the others?